packages/hmr-plugin/src/internal/hmr-state-context-factory.ts

```diff
@@ -21,6 +21,7 @@ export class HmrStateContextFactory<T, S> {
    */
   public createStateContext(): StateContext<S> {
     return {
+      abortController: new AbortController(),
```

Comment:
Not sure if this makes sense here, but we could just return:
```suggestion
      abortSignal: new AbortController().signal,
```